### ISSUE TITLE
Error installing cmake with ansible community.general.homebrew module

### ISSUE THREAD

**Issue by skeletony007** [skeletony007/ansible-roles#1]
### Error message

```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Warning: Treating cmake as a formula. For the cask, use homebrew/cask/cmake or specify the `--cask` flag."}
```

### To reproduce:

Include the [cmake role] in a playbook, for example:

```
ansible-playbook <(printf '%s' '---
- hosts: localhost
  roles:
    - role: cmake
')
```

[cmake role]: https://github.com/skeletony007/ansible-roles/tree/f0759f19fff61f804caedeb7ec8efc456826026d/cmake

**Comment by skeletony007** [comment]
This seems to have resolved itself. Upstream <https://github.com/ansible-collections/community.general> (or Homebrew itself) has fixed this issue.

**Comment by skeletony007** [comment]
I have reverted the temporary fix (commit: 06cd050b1bceda6c61e447d8d88da7a664b30c92) without error. I will mark this as closed.

Anyone with this issue should ensure they are running the latest version of Ansible and Homebrew.
